src/languages/stan.js

```diff
@@ -484,14 +325,14 @@ export default function(hljs) {
 
   const INCLUDE = {
     className: 'meta',
-    begin: /^#include\b/,
+    begin: /#\s*[a-z]+\b/,
```

Comment:
This seems wrong. The contains string rule only make sense for SOME directives, likely not all... so this would have to be much more nuanced if you wished to make it generic, no?

Reply:
I'm not sure, this seems to work for all my tests

Reply:
What is the full list of # directives Stan has?

Reply:
so #include can include anything really as long as the cpp compiler can pick it up. This includes .cpp, .stan. stanfunctions endings.

Reply:
I'm not taking about `#include`, but since you changed it to `#whatever` (but yet didn't touch the string contains rules) so I'm asking what directives Stan supports - or is this just using the C pre-compiler?

```
#ifdef SUPER_MODE
```

For example, SUPER_MODE here is not necessarily a string... 

Was this your intention? What were you trying to accomplish with this change?

Reply:
got it, updated to put include back in